engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -104,15 +110,16 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
 
   when (NotStarted) {
     case Event(RunWorkflow, data) =>
-      log.info(s"$Tag: Launching workflow")
-      val submitMessage = SubmitWorkflowCommand(source)
-      workflowManager ! submitMessage
+      log.info(s"$Tag: Submitting workflow")
+      workflowStore ! SubmitWorkflow(source)
```

Comment:
I had it that way first and then changed it. I can't remember why so it's impossible for me to say if the reason was good. If I remember in the near future I'll pipe up